podpac/core/node.py

```diff
@@ -544,7 +544,7 @@ def wrapper(self, coordinates, output=None):
             self._from_cache = False
 
         # transpose data to match the dims order of the requested coordinates
-        order = [dim for dim in coordinates.dims if dim in data.dims]
+        order = [dim for dim in coordinates.idims if dim in data.dims]
```

Comment:
@jmilloy this will help if the requested coordinates are Dependent. There is still an uncertain case when an `output` is provided but is asymmetric (Depedent/Not-Dependent) from the request or native coordinates